shopinvader/models/shopinvader_backend.py

```diff
@@ -435,3 +437,58 @@ def _normalize_name(name):
     def _onchange_name(self):
         if self.name and not self.tech_name:
             self.tech_name = self._normalize_name(self.name)
+        auth_api_key = getattr(request, "auth_api_key", None)
```

Comment:
@sebastienbeau, @rousseldenis   PLZ REVERT !!!!!! "auth_api_key" should never be used here!

Reply:
Here is the fix : https://github.com/shopinvader/odoo-shopinvader/pull/738